app/views/application/_notebook_tile_small.slim

```diff
@@ -16,7 +16,7 @@ div.info.square
         '  Notebook description:
       a.nounderline.notebook-description.tooltips href="#{notebook_path(notebook)}" title="#{notebook.description}"
         i.glyphicon.glyphicon-info-sign aria-hidden="true"
-        span.sr-only ==notebook.description
+        span.sr-only ==strip_tags(notebook.description)
```

Comment:
This is for screen readers only so wanted to strip tags but not "render" it to possibly break things